### PR TITLE
Add support Mapbox api 4 and https://github.com/Norkart/Leaflet-MiniMap

### PR DESCRIPTION
**Pull request by elesdoar**
Add support Mapbox Api v4 and https://github.com/Norkart/Leaflet-MiniMap

See: https://github.com/elesdoar/angular-leaflet-directive/blob/master/examples/0404-controls-minimap-example.html

### CODE REVIEW

**Comment by nmccready** [comment]
So overall mapbox api v3 is backwards compat? Also can you rebase and make the commit message in the format of angular commit conventions. https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md

**Comment by nmccready** [comment]
Buller?

**Comment by elesdoar** [comment]
Sorry, I've been busy, I am returning to work tomorrow. I'll modify the commit.

**Comment by nmccready** [comment]
Cool no worries that's just a ping 😯

**Comment by elesdoar** [comment]
:+1:

**Comment by elesdoar** [comment]
**Commit message:**

feat(control): Add minimap control option

Added new minimap option to controls:
- Add suport for https://github.com/Norkart/Leaflet-MiniMap control 
- Add support for mapbox v4

Is it ok?

File: dist/angular-leaflet-directive.js [hunk 1]
```diff
@@ -2848,6 +2884,35 @@ angular.module("leaflet-directive").directive('controls', ["$log", "leafletHelpe
...
+                                        map.addControl(minimapControl);
+                                        map.off('moveend', moveend);
+                                    };
+                                    map.on('moveend', moveend);
+                                } else {
+                                    var minimapControl = new L.Control.MiniMap(layer, controls.minimap);
+                                    map.addControl(minimapControl);
+                                }
+                            } else {
+                                $log.warn('[AngularJS - Leaflet] Layer could not be created.');
```
Comment (nmccready) [inline dist/angular-leaflet-directive.js]: The `[AngularJS - Leaflet]` string is in leafletHelpers.errorHeader. You could extend that string locally to `errorHeader + '  Layer'.

Comment (elesdoar) [inline dist/angular-leaflet-directive.js]: OK, I forgot it.

Comment (elesdoar) [inline dist/angular-leaflet-directive.js]: Fixed :+1: :smile:

**Comment by elesdoar** [comment]
Hi, Is it ok? :flushed:

**Comment by nmccready** [comment]
Yeah my bad, been busy w with work (on travel).

**Comment by elesdoar** [comment]
Don't worry!,.. Be huggies... :smile: :sunglasses:

**Pull request merged**